src/vs/workbench/browser/parts/editor/tabsTitleControl.ts

```diff
@@ -881,14 +882,13 @@ export class TabsTitleControl extends TitleControl {
 	private computeTabLabels(): void {
 		const { labelFormat } = this.accessor.partOptions;
 		const { verbosity, shortenDuplicates } = this.getLabelConfigFlags(labelFormat);
-
 		// Build labels and descriptions for each editor
 		const labels = this.group.editors.map(editor => ({
 			editor,
 			name: editor.getName(),
 			description: editor.getDescription(verbosity),
 			title: withNullAsUndefined(editor.getTitle(Verbosity.LONG)),
-			ariaLabel: editor.isReadonly() ? localize('readonlyEditor', "{0} readonly", editor.getTitle(Verbosity.SHORT)) : editor.getTitle(Verbosity.SHORT)
+			ariaLabel: computeEditorAriaLabel(editor, this.group, this.editorGroupService.count)
```

Comment:
Are we keeping the `isReadonly` semantics?

Reply:
No. Since the readonly is part of tha title now as (read-only) it already gets read out. So i removed it on purpose to not have duplication.